applications/nrf5340_audio/CMakeLists.txt

```diff
@@ -96,7 +97,7 @@ if ((CONFIG_AUDIO_DFU EQUAL 1) OR (CONFIG_AUDIO_DFU EQUAL 2))
 
```

Comment:
Should move to here.
file(TO_CMAKE_PATH ${ZEPHYR_NRF_MODULE_DIR}/lib/bin/bt_ll_pcft/bin CMAKE_STYLE_LL_PCFT_BIN)